build.sh

```diff
@@ -13,7 +13,7 @@ echo "Building BitBetter for BitWarden version $BW_VERSION"
 
 cp "$DIR/.keys/cert.cert" "$DIR/src/bitBetter/.keys"
 
-docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/core/sdk:3.1 sh build.sh
+docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/sdk sh build.sh
```

Comment:
can we fix the version to :5.0 if we are using netcoreapp5.0 in the other files?

```suggestion
docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/sdk:5.0 sh build.sh